addons/mail/static/src/core/web_portal/thread_service_patch.js

```diff
@@ -0,0 +1,99 @@
+/* @odoo-module */
+
+import { ThreadService } from "@mail/core/common/thread_service";
+
+import { rpc } from "@web/core/network/rpc";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+import { Record } from "@mail/core/common/record";
+
+patch(ThreadService.prototype, {
+    /**
+     * @param {import("models").Thread} thread
+     * @param {['activities'|'followers'|'attachments'|'messages'|'suggestedRecipients']} requestList
+     */
+    async fetchData(
```

Comment:
To be double checked, but shouldn't `fetchData` fully be moved to `chatter` feature? Is it even used in another context?

Reply:
The only use is in `chatter`.

Reply:
Then we should move it to `/chatter` instead of `/core`